schemas/questionnaire_v1.json

```diff
@@ -62,7 +62,26 @@
     },
     "metadata": {
       "type": "object",
-      "required": ["user_id", "period_id"]
+      "required": [
+        "user_id",
+        "period_id"
+      ],
+      "patternProperties": {
+        "^.*$": {
+          "type": "object",
+          "properties": {
+            "validator": {
+              "type": "string",
+              "enum": [
+                "date",
+                "string",
+                "object"
```

Comment:
What does the object validator check for? I'm not saying its wrong just wondering which metadata values would be passed as an object, do you have an example?